package.json

```diff
@@ -38,6 +38,7 @@
     "clsx": "^2.1.1",
     "graphql": "^14 || ^15 || ^16",
     "graphql-request": "^6.1.0",
+    "jsbi": "^4.3.0",
```

Comment:
Very intersting, i believe as well a library of this kind can help us. But before we merge it, @0xAlec can you take me into the trade-off in adding this libraries. Usual things like:
- how many Kb the library is
- what's the state of being manteined
- what risk are we accepting to take by adding this library
- and anything else?

Reply:
- 568 kB unpacked size
- maintained by Google Chrome team
- not sure, we can use alternatively use native BigInts at expense of compatibility for older browsers

Reply:
`568 kB` is quite a lot, I wonder how much will be minified.